base/files/file.h

```diff
@@ -22,7 +22,11 @@ struct stat;
 
 namespace base {
 
+#if defined(STARBOARD)
+using stat_wrapper_t = struct ::stat;
```

Comment:
Why do we need this special global syntax for Starboard?

Reply:
If not using the global namespace, we will run into below error here:
../../base/files/file_starboard.cc(62): error C2027: use of undefined type 'base::stat'
C:\runner\_work\cobalt\cobalt\base/files/file.h(26): note: see declaration of 'base::stat'